README.md

```diff
@@ -12,6 +12,7 @@ It helps developers to:
 -   `deploy` applications to a cloud provider
 -   `monitor` application already running on a cloud provider
 -   `doctor` verify the minimal requirements for the Sunodo CLI commands
+-   `hash` prints out image hash generated by the build command
```

Comment:
It would be nice to be more specific that this hash is the `templateHash` of the snapshot, since the container image also has a hash and this could led to confusion.

Not sure about the best wording here, if `tempalteHash` or just `hash` or `snapshot hash`.

```suggestion
-   `hash` prints out Cartesi Machine tempalteHash generated by the build command
```

Reply:
Agree, just you added a typo there